A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/ATSU/A320_Neo_CDU_AOC_MessagesReceived.js

```diff
@@ -23,6 +23,10 @@ class CDUAocMessagesReceived {
         }
         mcdu.clearDisplay();
 
+        mcdu.refreshPageCallback = () => {
+            this.ShowPage(mcdu);
+        };
```

Comment:
Note: this will return to the first page, showing the new message. I think this would usually be desirable but it's debatable.